pkg/logconfig/config.go

```diff
@@ -31,3 +36,22 @@ const (
 	// Webhook is the name of the override key used inside of the logging config for Webhook Controller.
 	Webhook = "webhook"
 )
+
+func ConfigName() string {
```

Comment:
nit, this might be clearer if it was: ConfigMapName rather than ConfigName?

Reply:
Done